astronomy/trappist_dynamics_test.cpp

```diff
@@ -1029,8 +1029,8 @@ class TrappistDynamicsTest : public ::testing::Test {
       Sign const xy_displacement_derivative_sign(xy_displacement_derivative(t));
       if (relative_dof.displacement().coordinates().z > 0.0 * Metre &&
           last_t &&
-          xy_displacement_derivative_sign == Sign(1) &&
-          last_xy_displacement_derivative_sign == Sign(-1)) {
+          xy_displacement_derivative_sign.is_positive() &&
+          last_xy_displacement_derivative_sign == Sign::Negative()) {
```

Comment:
`is_negative`?

Reply:
No, that's an optional sign on the left hand side.